src/extension/app-schema/app-schema-indexes-test/src/test/java/org/geoserver/test/onlineTest/ComplexIndexesTest.java

```diff
@@ -150,17 +140,15 @@ public static void beforeClass() throws Exception {
     public void beforeTest() {
         // instantiate WFS 1.1 xpath engine
         Pair<String, String> stationsNamespace = Pair.of("st", STATIONS_NAMESPACE);
-        WFS11_XPATH_ENGINE =
-                buildXpathEngine(
-                        Pair.of("wfs", "http://www.opengis.net/wfs"),
-                        Pair.of("gml", "http://www.opengis.net/gml"),
-                        stationsNamespace);
+        WFS11_XPATH_ENGINE = buildXpathEngine(
```

Comment:
Glancing over quickly I noticed this...  looks like reformatting... has palantir been udpated?

Reply:
Hi Andrea, indeed it looks like pure formatting changes to some files.
I can't figure out why though. 
* The palantir formatter config hasn't changed.
* The maven-compiler-plugin version did change, but tried on a separate branch with only 3ad815fe13a5447c3990b74aaed34d8fac7af0da, which only changes pom.xml files, and `mvn spotless:apply -Prelease,communityRelease` doesn't change anything
* Then checked out 102ae3bdbb7e949cc97420dcdc59fe61f3424181, and same story.
* Same thing with the current head commit (702fb0941b91383e767488b5648302b82869ed57)

I'm suspecting Eclipse did the changes but the palantir formatter is ok with it.

Reply:
With @jodygarnett we identified the culprit being some modules escaped the formatting. I've created https://github.com/geoserver/geoserver/pull/8635 and will rebase this pr once its applied

Reply:
@jodygarnett @aaime in the meantime, I've removed those purely formatting changes from this pull request